<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>179</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>423</width>
    <height>179</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Edit Bookmark</string>
  </property>
  <widget class="QDialogButtonBox" name="button_box">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>231</width>
     <height>32</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>800000</width>
     <height>800000</height>
    </size>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
   </property>
  </widget>
  <widget class="QLabel" name="name_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #fafafa</string>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name_textbox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit:disabled {
	background-color: rgb(197, 197, 197);
}

QLineEdit {
	background-color: rgb(250, 250, 250);
	border-radius: 4px;
	border: 0;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="link_textbox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit:disabled {
	background-color: rgb(197, 197, 197);
}

QLineEdit {
	background-color: rgb(250, 250, 250);
	border-radius: 4px;
	border: 0;
}</string>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="link_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #fafafa</string>
   </property>
   <property name="text">
    <string>Link:</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>451</width>
     <height>261</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>10000</width>
     <height>10000</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(47, 47, 47);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <zorder>frame_3</zorder>
  <zorder>button_box</zorder>
  <zorder>name_label</zorder>
  <zorder>name_textbox</zorder>
  <zorder>link_textbox</zorder>
  <zorder>link_label</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
